packages/analytics-js/__tests__/components/configManager/commonUtil.test.ts

```diff
@@ -297,14 +302,28 @@ describe('Config Manager Common Utilities', () => {
       expect(state.serverCookies.isEnabledServerSideCookies.value).toBe(true);
       expect(state.serverCookies.dataServiceUrl.value).toBe('https://www.test-host.com/rsaRequest');
     });
-    it('should set isEnabledServerSideCookies to false if provided setCookieDomain load option is different from current domain', () => {
+
+    it('should set isEnabledServerSideCookies to false if provided setCookieDomain load option is different from current domain and sameDomainCookiesOnly option is not set', () => {
       state.loadOptions.value.useServerSideCookies = true;
       state.loadOptions.value.setCookieDomain = 'test-host.com';
 
       (getDataServiceUrl as jest.Mock).mockImplementation(originalGetDataServiceUrl);
       updateStorageStateFromLoadOptions(mockLogger);
 
       expect(state.serverCookies.isEnabledServerSideCookies.value).toBe(false);
+      expect(mockLogger.warn).toHaveBeenCalledWith(
+        'ConfigManager:: The server side cookie setting feature has been disabled because provided cookie domain (test-host.com) is not matching with current domain (www.test-host.com).',
+      );
+    });
+    it('should set isEnabledServerSideCookies to true if provided setCookieDomain load option is different from current domain and sameDomainCookiesOnly option is set', () => {
```

Comment:
Please add a blank line here too to improve the readability.

Reply:
done